### ISSUE TITLE
Some ideas.

### ISSUE THREAD

**Issue by Arcachofo** [SimulIDE/SimulIDE#32]
Hi guys.

Nice to see there are people actually working on this fork.

I want to share some ideas about this project:

**- Project name:**
If this is a real fork i think it would be better using a different name or a variation of the original name, showing that this project is taking it's own path. 
If you will just retrieve the code from the original project and make a good repo, then it looks right to use the same name.

**- Source code**
You can get the source code and available builds here: https://www.simulide.com/p/downloads.html
If you need help with some issue that might appear while you are changing things, just let me know.
There is a forum and some resources at https://www.simulide.com

**- Gpsim and simavr:**
Gpsim and simavr dependencies come bundled into simulide source code for a reason.
It would be possible to outsource them with some work, but the direction i'm taking in simulide is integrating them more and more, even integrating both together.
So depending what is your direction, outsourcing could be a good or a very bad idea.

I really wish that this project becomes a real fork and provide new solutions. 
But even if it is just a repo for the original project, it will be useful. I’ve already learned several things.

**Comment by Mihaylov93** [comment]
Hello, this fork is still ongoing we started basically from scratch, no one has experience with the project. If you check the branch wip/clean there is work done about the naming, SimulIDE CE (community edition).

For the outsourcing, we have it as an issue, we haven't decided yet but if you checked latest commits, maybe oursorcing wont be as straightforward as we tought as we have made modifications on avrsim for windows compatibility.

**Comment by Arcachofo** [comment]
Yes, i see that you have been working just a few days and got quite a few things done.

Outsourcing is not straightfordward at all. These are very modified versions, the original gpsim and simavr wont work with simulide (in it's current state).

You could separate them as external projects, but even this might need some changes (specially gpsim).

Using the original libraries would require quite a few changes in simulide and finding new solutions to those issues that caused bringing them to simulide.
If this helps, in sourceforge svn repo you can find branches that used original gpsim and simavr:
- Versions below 0.3.X used original gpsim.
- Versions below 0.1.X used original simavr.

But there are many things not working in those versions.
There are reasons why it is like it is now.

I wouldn't get into this until you get an idea of how it is working now, then make a decission.
If you decide to outsource i could give some insight about where are the "hotspots".

**Comment by Mihaylov93** [comment]
> If you decide to outsource i could give some insight about where are the "hotspots".

That will be great!

I would say its early for that, for now what we have worked on were QoL.

**Comment by albfan** [comment]
Honestly, the name for this org/project is really confusing.

If there's no link with real project I strongly suggest to share the organization and change the project name.

openSimulIDE, SimulIDEfork?

Adding on README the areas where this project extends or derive the original project is a must.

And I don't see why it cannot be a branch on original project or a set of issues and PR
